tusclient/client.py

```diff
@@ -16,15 +16,21 @@ class TusClient:
             along with every request made by the cleint to the server. This may be used to set
             authentication headers. These headers should not include headers required by tus
             protocol. If not set this defaults to an empty dictionary.
-
+        - client_cert (tuple):
+            Path of PEM encoded client certitifacate and optionally path to PEM encoded
+            key file. The PEM encoded key of the certificate can either be included in the
+            certificate itself or be provided in a seperate file.
+            Only unencrypted keys are supported!
     :Constructor Args:
         - url (str)
         - headers (Optiional[dict])
+        - client_cert (Optional[Tuple[str, str]])
```

Comment:
The type indicates that it wants a tuple with two elements. How would this work if I just want to supply a PEM encoded client certificate and no PEM encoded key file?

Reply:
The type hint is incorrect. It should be the same as the sessions library: `str | tuple[str, str]`